lib/pushmi_pullyu/cli.rb

```diff
@@ -160,15 +160,21 @@ def run_tick_loop
     while PushmiPullyu.server_running?
       # Preservation (TODO):
       item = @queue.wait_next_item
-      logger.debug(item)
-      # 3. Retrieve GenericFile data in fedora
-      # 4. creation of AIP
-      # 5. bagging and tarring of AIP
-      # 6. Push bag to swift API
-      file_to_deposit = './pushmi_pullyu'
```

Comment:
yes, I removed that  I thought we will need to change that anyways... can add it back